### PR TITLE
Feature/Set up @Output for click in activities table component

### PR DESCRIPTION
**Pull request by Yordaniss**
ISSUE #3836

* eliminate router from lib component
* added onclicked event into activities page
* added functionality into html

### CODE REVIEW

File: apps/client/src/app/pages/portfolio/activities/activities-page.component.ts [hunk 1]
```diff
@@ -255,6 +255,16 @@ export class ActivitiesPageComponent implements OnDestroy, OnInit {
     });
   }
 
+  public onClickActivity(aActivity: Activity) {
```
Comment (dtslvr) [inline apps/client/src/app/pages/portfolio/activities/activities-page.component.ts:258]: Please move up ⬆️ to keep the alphabetical order of the methods

Comment (dtslvr) [inline apps/client/src/app/pages/portfolio/activities/activities-page.component.ts:258]:
```
public onClickActivity({ SymbolProfile }: Activity) {
```

File: apps/client/src/app/pages/portfolio/activities/activities-page.component.ts [hunk 2]
```diff
@@ -255,6 +255,16 @@ export class ActivitiesPageComponent implements OnDestroy, OnInit {
...
+    this.router.navigate([], {
+      queryParams: {
+        dataSource: aActivity.SymbolProfile.dataSource,
+        symbol: aActivity.SymbolProfile.symbol,
+        holdingDetailDialog: true
```
Comment (dtslvr) [inline apps/client/src/app/pages/portfolio/activities/activities-page.component.ts:263]: Please move up (alphabetical order)

File: libs/ui/src/lib/activities-table/activities-table.component.ts [hunk 3]
```diff
@@ -105,6 +105,7 @@ export class GfActivitiesTableComponent
   @Output() pageChanged = new EventEmitter<PageEvent>();
   @Output() selectedActivities = new EventEmitter<Activity[]>();
   @Output() sortChanged = new EventEmitter<Sort>();
+  @Output() activityClicked = new EventEmitter<AssetProfileIdentifier>();
```
Comment (dtslvr) [inline libs/ui/src/lib/activities-table/activities-table.component.ts:108]: Please move up ⬆️ to keep the alphabetical order

File: libs/ui/src/lib/activities-table/activities-table.component.ts
Comment (dtslvr) [inline libs/ui/src/lib/activities-table/activities-table.component.ts:1]: Please remove unused router:

```
❌ import { Router, RouterModule } from '@angular/router';
```
```
❌ RouterModule
```
```
❌ private router: Router
```

**Review by dtslvr — APPROVED** [review]
Thanks for your contribution, @Yordaniss! 👍🏻
I have added a few comments.

**Review by dtslvr — CHANGES_REQUESTED** [review]
Thanks for your contribution, @Yordaniss! 👍🏻
I have added a few comments.

File: libs/ui/src/lib/activities-table/activities-table.component.ts
Comment (Yordaniss) [reply libs/ui/src/lib/activities-table/activities-table.component.ts:1]: done

**Review by dtslvr — APPROVED** [review]
Thanks again for your contribution, @Yordaniss! 🙂
I have made some final adjustments. Ready to merge now.

**Comment by Yordaniss** [comment]
You welcome @dtslvr was a pleasure to work on this issue

**Pull request merged**